### PR TITLE
Removed unnecessary Silverlight preprocessor directives and classes

### PR DESCRIPTION
**Pull request by gpetrou**

### CODE REVIEW

File: Tests/RowanTest.Common/Runner.cs [hunk 1]
```diff
@@ -80,10 +78,6 @@ public static class Runner {
         static Runner() {
             if (System.IntPtr.Size == 8) _Is64 = true;
 
-#if SILVERLIGHT
-            _IsSilverlight = true;
```
Comment (slozier) [inline Tests/RowanTest.Common/Runner.cs]: Maybe get rid of the variable definition as well?

Comment (gpetrou) [inline Tests/RowanTest.Common/Runner.cs]: Sure.

**Comment by gpetrou** [comment]
OK, but in that case are you squashing the commits when you merge?
Personally, I prefer having one commit per PR.

**Comment by slozier** [comment]
Yes, normally we squash and merge.

**Pull request merged**